pkg/cmd/cli/cmd/edit.go

```diff
@@ -23,14 +22,27 @@ import (
 	"github.com/openshift/origin/pkg/util/jsonmerge"
 )
 
+type EditOptions struct {
```

Comment:
@mfojtik I agreed with you, didn't I? Let's do it right here AND I proposed them to fix it in other places later.